src/ethereum-canister/src/helios.rs

```diff
@@ -35,6 +37,11 @@ pub(crate) async fn start_client(
         bail!("Client already started");
     }
 
+    let network = match network {
```

Comment:
Can we have From and Into instead?

Reply:
We can but I'm not sure if I like adding `helios_config` as a dependency in `interface`.